source/_docs/guides/drupal-8-simplesamlphp.md

```diff
@@ -0,0 +1,264 @@
+---
+title: How to Implement SimpleSAMLphp as idP for Drupal 8 on Pantheon 
+description: Setting up an identity provider on Drupal 8 using SimpleSAMLphp.
+draft: true
+tags: [code, drupal-8]
+categories: [drupal, developing]
+contributors: Javi-er
+---
+
+There are a number of articles detailing how to implement the SAML 2.0 protocol, but if you want to host your site on Pantheon and use Drupal 8 as your identity provider, there are a few details to take in account.
+<div class="alert alert-info">
+<h4 class="info">Note</h4><p markdown="1">The following tutorial assumes that you are on a linux based computer or a Mac. Also that your project is running on Apache.
+</p>
+</div>
+
+## Before You Begin
+
+Before starting with this guide, make sure you have the following:
+
+ - Have a working Drupal 8 site.
+ - Download the latest version of [SimpleSAMLphp](https://simplesamlphp.org/download).
+ - Download the Drupal module [saml_idp](https://github.com/Javi-er/saml_idp) (you’ll notice that this repo is a fork of [this one](https://github.com/bradjones1/saml_idp) this is because there were some changes necessary to make it work which are [described here](https://www.drupal.org/project/saml_idp/issues/3013357).)
+ 
+## Getting started
+
+Your project structure and configuration needs to be tweaked so it's recognized by Pantheon when deployed to the server.
+
+### Setting up SAML Idp module
+By now you should have SAML Idp module on your Drupal 8 site, go to the admin interface and enable it now.
+
+After this create the subdirectory ``/simplesamlphp/drupalauth`` and create an empty file with the name `default_enable` in that directory in order to enable the Drupal 8 integration. You can also do this with Drush executing
+
+``` bash
+drush ev 'Drupal\saml_idp\Install::postInstall()'
+```
+
+### Setting up SimpleSAMLphp
+
+1.  Place your Drupal 8 projected inside a ``web`` [nested docroot](https://pantheon.io/docs/nested-docroot).
+
+2.  Place SimpleSAMLphp a folder called ``/private`` 
+
+3.  Create a symlink on your project root from ``/simplesaml`` to ``/private/simplesamlphp/www``
+    ``` bash
+    $ ln -s ./private/simplesamlphp/www ./simplesaml
+    ```
+    
+    Your project structure should be as follow:
+    
+    ``` markdown
+    project
+    ├── pantheon.yml   
+    ├── ...
+    └─── private
+    │   │
+    │   └─── simplesamlphp
+    │       └─── ...
+    └─── simplesamlphp -> ./private/simplesamlphp/www  
+    |
+    └───web
+        ├── index.php
+        └── ...
+    ```
+    
+4.  Add a virtual host for your project
+    Edit your Apache virtual host confiuguration and add the following:
+    You can see your current virtual host configuration with 
+    ``` bash 
+    $ /usr/sbin/httpd -S
```

Comment:
So this guide requires that Apache be running on your local computer?

Additionally, the binary file for Apache differs based on the OS - if this command can be replaced with an `apachectl` command it would be more universal.

Reply:
I removed all Apache references except by an example so it's server agnostic now.